File: Data Structures & Algorithms/two-integer-sum/submission-0.go
func twoSum(nums []int, target int) []int {
	var result = make([]int, 2)
	if len(nums) > 1{
		for i := range nums{
			for j:=1; j < len(nums); j++{
				if i == j{
					continue
				}
				if nums[i] + nums[j] == target {
					result[0], result[1] = i, j
					return result
				}
			}
		}
	}

	return result
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:

        if len(numbers) == 2:
            return [1, 2]

        map = {}

        for i in range(len(numbers)):
            found_it = map.get(target - numbers[i], 0)
            if found_it:
                # print("found_it at index ", found_it, " and value ", target-numbers[i])
                return [found_it, i+1]
            map[numbers[i]] = i+1
            # print(i, " : ", map)

        return []
        